distributed/deploy/ssh.py

```diff
@@ -116,12 +121,20 @@ async def start(self):
                 set_env,
                 self.remote_python,
                 "-m",
-                self.worker_module,
+                "distributed.cli.dask_spec",
```

Comment:
In some other implementations we also serialize the local config and pass it along in an environment variable. Might be worth doing the same here.

https://github.com/dask/dask-cloudprovider/blob/fcc6cf37933d5098d3f0a9ed55e3437ca3cd6e98/dask_cloudprovider/generic/vmcluster.py#L33-L35

Reply:
Looks like that might already be happening a few lines above?

https://github.com/dask/distributed/blob/31afb54cf02141050ae1cc7cbd7c3919557637fe/distributed/deploy/ssh.py#L93-L106

Reply:
Awesome! I totally missed that, didn't expand the diff properly.